testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -44,8 +45,27 @@ pub struct ClusterSwarmKube {
 
 impl ClusterSwarmKube {
     pub async fn new() -> Result<Self> {
-        let result = Config::infer().await;
-        let config = result.map_err(|e| format_err!("Failed to load config: {:?}", e))?;
+        // This uses kubectl proxy locally to forward connections to kubernetes api server
+        thread::spawn(|| {
+            info!("Starting kubectl proxy");
+            Command::new("/usr/local/bin/kubectl")
```

Comment:
Curious - why do you need spawn thread for it?

Reply:
I shouldn't need it. I'll fix it.